x-pack/plugins/cases/server/services/notifications/templates/assignees/renderer.ts

```diff
@@ -79,6 +80,7 @@ export const assigneesTemplateRenderer = async (
         ? `${caseData.attributes.description.slice(0, DESCRIPTION_LIMIT)}...`
         : caseData.attributes.description,
     url: caseUrl,
+    currentYear: moment().year(),
```

Comment:
I think it is ok if we use `new Date().getUTCFullYear()` to be consistent with different timezones.